src/client/components/applayout/AppLayout.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import HeaderLayout from './HeaderLayour';
+import FooterLayout from './FooterLayout';
+import ContentLayout from './ContentLayout';
+export class AppLayout extends React.Component {
+  render() {
+    return (
+      <div className={'container-fluid'}>
```

Comment:
String props can be passed directly without brackets/expressions `<div className='container-fluid'>`